src/Chapter21.Tests/Listing21.04.Tests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter21.Listing21_04.Tests
+{
+    [TestClass]
+    public class ProgramTests
+    {
+        [TestMethod]
+        public void MainTest()
+        {
+            const string expected = @"ERROR: AggregateException:
+  UnauthorizedAccessException - Attempted to perform an unauthorized operation.
+  UnauthorizedAccessException - Attempted to perform an unauthorized operation.
+  UnauthorizedAccessException - Attempted to perform an unauthorized operation.
+  UnauthorizedAccessException - Attempted to perform an unauthorized operation.
+  UnauthorizedAccessException - Attempted to perform an unauthorized operation.
+  UnauthorizedAccessException - Attempted to perform an unauthorized operation.
+  UnauthorizedAccessException - Attempted to perform an unauthorized operation.
+  UnauthorizedAccessException - Attempted to perform an unauthorized operation.
+  UnauthorizedAccessException - Attempted to perform an unauthorized operation.";
```

Comment:
We probably want to catch and check this exception instead of checking the console output